services/galley/src/Galley/Types/TeamEvents.hs

```diff
@@ -0,0 +1,40 @@
+{-# LANGUAGE DataKinds         #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeOperators     #-}
+
+module Galley.Types.TeamEvents
+    ( teamCreate
+    , teamUpdate
+    , teamDelete
+    ) where
+
+import Control.Monad.IO.Class
+import Data.ByteString.Conversion hiding (fromList)
+import Data.Int
+import Data.Id
+import Data.Time.Clock (getCurrentTime)
+import Data.Time.Zones.Internal
+import Prelude hiding (head, mapM)
+
+import Proto.Galley.Types.TeamEvents
+
+teamCreate :: TeamId -> UserId -> IO TeamEvent
+teamCreate tid uid = do
+    now <- utcTimeToInt64 <$> liftIO getCurrentTime
```

Comment:
Instead of requiring a new library, how about using [utcTimeToPOSIXSeconds](https://hackage.haskell.org/package/time-1.8.0.2/docs/src/Data.Time.Clock.POSIX.html#utcTimeToPOSIXSeconds) which is part of time?

```
> n <- getCurrentTime
> n
2017-07-26 12:52:28.373924 UTC
> utcTimeToInt64 n
1501073548
> round (utcTimeToPOSIXSeconds n) :: Int64
1501073548
```